.github/CODEOWNERS

```diff
@@ -641,8 +641,6 @@ pep-3154.txt  @pitrou
 pep-3155.txt  @pitrou
 pep-3156.txt  @gvanrossum
 # ...
-pep-3333.txt  @pjeby
```

Comment:
Should we keep 3333 here (commented) to signal the PEP exists?

A

Reply:
Good point, done